parsimonious/tests/test_grammar.py

```diff
@@ -139,9 +139,25 @@ def test_optional(self):
                                            Node(Literal("howdy"), howdy, 0, 5)]))
 
 
+def function_rule(text, pos):
```

Comment:
Could use a docstring here; I didn't know what was going on until I got down to `method_rule()` and saw the pattern.